packages/lesswrong/server/reviewVoteUpdate.ts

```diff
@@ -300,3 +303,68 @@ export async function createVotingPostHtml () {
 //   AND "postedAt" < '2024-01-01'
 //   AND "positiveReviewVoteCount" > 0
 // ORDER BY "reviewVoteScoreHighKarma" DESC
+
+const createReviewWinners = async () => {
+  const posts = await Posts.find({
+    postedAt: {
+      $gte:moment(`${REVIEW_YEAR}-01-01`).toDate(), 
+      $lt:moment(`${REVIEW_YEAR+1}-01-01`).toDate()
+    }, 
+    finalReviewVoteScoreAllKarma: {$gte: 1},
+    reviewCount: {$gte: 1},
+    positiveReviewVoteCount: {$gte: 1}
+  }, {sort: {finalReviewVoteScoreHighKarma: 1}, limit: 1}).fetch()
+
+  const tagIdsFromPosts = posts.map(post => Object.keys(post.tagRelevance))
+  const tagIds = [...new Set(tagIdsFromPosts.flat())]
+  const tags = await Tags.find({_id: {$in: tagIds}}).fetch()
+
+  const adminContext = createAdminContext();
+
+  const coreTags: DbTag[] = await Tags.find({name: {$in: ["Rationality", "World Modeling", "World Optimization", "Practical", "AI"]}}).fetch() ?? []
+
+  const aiStrategyTags = await Tags.find({name: {$in: ["AI Governance", "AI Timelines", "AI Takeoff", "AI Risk"]}}).fetch() ?? []
+
+  const getPostCoreTag = (post: DbPost) => {
+    const tagRelevance = post.tagRelevance
+    const coreTagsOnPost = coreTags.filter(tag => tagRelevance[tag._id] > 0)
+    const postHasAIStrategyTag = aiStrategyTags.some(tag => tagRelevance[tag._id] > 0)
+    const coreTagsMostRelevant = coreTagsOnPost.sort((a,b) => tagRelevance[b._id] - tagRelevance[a._id])[0]
+    if (coreTagsMostRelevant?.name === "AI") {
+      return postHasAIStrategyTag ? "ai strategy" : "ai safety"
+    } else {
+      return tagToCategory[coreTagsMostRelevant?.name]
+    }
+  }
+
+  const tagToCategory: Record<string,ReviewWinnerCategory> = {
+    "Rationality": "rationality",
+    "World Modeling": "modeling",
+    "World Optimization": "optimization",
+    "Practical": "practical"
+  }
```

Comment:
declare this outside of `createReviewWinners` (and maybe name `tagToReviewWinnerCategory`?